core/scss/components/input/input.scss

```diff
@@ -131,39 +131,39 @@
       }
     }
 
-    &.error {
+    &.warning {
```

Comment:
Was this just a compilation change so that error was below warning? Not a big deal as we transition to the new input but we'd probably want to filter the possible classes to apply in the javascript instead of applying both classes if possible.

Reply:
yeah just to keep it below warning

Reply:
I agree that we should try to only apply one className with the JS if possible, however the order change should also be applied for the priority *IF* both classes exist on the same input.